sms_sponsorship/models/staff_notification_settings.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2018 Compassion CH (http://www.compassion.ch)
+#    @author: Quentin Gigon <gigon.quentin@gmail.com>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+
+from odoo import api, models, fields
+
+
+class StaffNotificationSettings(models.TransientModel):
+    """ Settings configuration for any Notifications."""
+    _name = 'staff.notification.settings'
+    _inherit = 'staff.notification.settings'
+
+    # Users to notify after Disaster Alert
+    new_partner_notify_ids = fields.Many2many(
+        'res.partner', string='SMS new partner')
```

Comment:
Add a relation name, because we have several Many2many fields pointing to res.partner. See https://www.odoo.com/documentation/10.0/reference/orm.html#relational-fields for field reference